<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(49, 47, 57);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="_move">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>45</y>
      <width>160</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 65, 84);
border-radius: 15px;
border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_rotate">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>175</y>
      <width>160</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 65, 84);
border-radius: 15px;
border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_scale">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>305</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 65, 84);
border-radius: 15px;
border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_description">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>520</y>
      <width>190</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 65, 84);
border-radius: 15px;
border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="_move_txt">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>50</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 65, 84);
color: rgb(221, 231, 220);</string>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QLabel" name="_rotate_txt">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>180</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 65, 84);
color: rgb(221, 231, 220);</string>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QLabel" name="_scale_txt">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>310</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 65, 84);
color: rgb(221, 231, 220);</string>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QLabel" name="_description_txt">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>530</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 65, 84);
color: rgb(221, 231, 220);</string>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_move_x">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>77</y>
      <width>120</width>
      <height>7</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    height: 5px;
    background: rgb(217, 217, 217);
    border-radius: 1px;
}

QSlider::handle:horizontal {
    background: rgb(177, 177, 177);
    width: 13px;
    margin: -3px 0;
    border-radius: 4px;
}

QSlider::horizontal {
    background-color: rgb(217, 217, 217);
}</string>
    </property>
    <property name="minimum">
     <number>-200</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_move_y">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>99</y>
      <width>120</width>
      <height>7</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    height: 5px;
    background: rgb(217, 217, 217);
    border-radius: 1px;
}

QSlider::handle:horizontal {
    background: rgb(177, 177, 177);
    width: 13px;
    margin: -3px 0;
    border-radius: 4px;
}

QSlider::horizontal {
    background-color: rgb(217, 217, 217);
}</string>
    </property>
    <property name="minimum">
     <number>-200</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_move_z">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>121</y>
      <width>120</width>
      <height>7</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    height: 5px;
    background: rgb(217, 217, 217);
    border-radius: 1px;
}

QSlider::handle:horizontal {
    background: rgb(177, 177, 177);
    width: 13px;
    margin: -3px 0;
    border-radius: 4px;
}

QSlider::horizontal {
    background-color: rgb(217, 217, 217);
}</string>
    </property>
    <property name="minimum">
     <number>-200</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_rotate_z">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>252</y>
      <width>120</width>
      <height>7</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    height: 5px;
    background: rgb(217, 217, 217);
    border-radius: 1px;
}

QSlider::handle:horizontal {
    background: rgb(177, 177, 177);
    width: 13px;
    margin: -3px 0;
    border-radius: 4px;
}

QSlider::horizontal {
    background-color: rgb(217, 217, 217);
}</string>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_rotate_y">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>230</y>
      <width>120</width>
      <height>7</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    height: 5px;
    background: rgb(217, 217, 217);
    border-radius: 1px;
}

QSlider::handle:horizontal {
    background: rgb(177, 177, 177);
    width: 13px;
    margin: -3px 0;
    border-radius: 4px;
}

QSlider::horizontal {
    background-color: rgb(217, 217, 217);
}</string>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_rotate_x">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>208</y>
      <width>120</width>
      <height>7</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    height: 5px;
    background: rgb(217, 217, 217);
    border-radius: 1px;
}

QSlider::handle:horizontal {
    background: rgb(177, 177, 177);
    width: 13px;
    margin: -3px 0;
    border-radius: 4px;
}

QSlider::horizontal {
    background-color: rgb(217, 217, 217);
}</string>
    </property>
    <property name="minimum">
     <number>-360</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_scale">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>335</y>
      <width>120</width>
      <height>7</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    height: 5px;
    background: rgb(217, 217, 217);
    border-radius: 1px;
}

QSlider::handle:horizontal {
    background: rgb(177, 177, 177);
    width: 13px;
    margin: -3px 0;
    border-radius: 4px;
}

QSlider::horizontal {
    background-color: rgb(217, 217, 217);
}</string>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="_moveX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 65, 84);
color: rgb(221, 231, 220);</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="_moveY">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>92</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 65, 84);
color: rgb(221, 231, 220);</string>
    </property>
    <property name="text">
     <string>y</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="_moveZ">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>114</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 65, 84);
color: rgb(221, 231, 220);</string>
    </property>
    <property name="text">
     <string>z</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="_rotateX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>201</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 65, 84);
color: rgb(221, 231, 220);</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="_rotateZ">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>245</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 65, 84);
color: rgb(221, 231, 220);</string>
    </property>
    <property name="text">
     <string>z</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="_rotateY">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>223</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 65, 84);
color: rgb(221, 231, 220);</string>
    </property>
    <property name="text">
     <string>y</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="filename">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>552</y>
      <width>175</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 65, 84);
color: rgb(221, 231, 220);</string>
    </property>
    <property name="text">
     <string>Filename: </string>
    </property>
   </widget>
   <widget class="QLabel" name="vertex_cnt">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>574</y>
      <width>175</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 65, 84);
color: rgb(221, 231, 220);</string>
    </property>
    <property name="text">
     <string>Count of vertex: </string>
    </property>
   </widget>
   <widget class="QLabel" name="poly_cnt">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>595</y>
      <width>173</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(59, 65, 84);
color: rgb(221, 231, 220);</string>
    </property>
    <property name="text">
     <string>Count of polygons: </string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>651</width>
      <height>651</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_1"/>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>640</y>
      <width>80</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 12px;
	border-style: solid;
	background-color: rgb(59, 65, 84);
	color: rgb(221, 231, 220);
}

QPushButton::hover {
	border-radius: 12px;
	border-style: solid;
	background-color: rgb(49, 55, 74);
	color: rgb(221, 231, 220);
}</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_file">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>640</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 10px;
	border-style: solid;
	background-color: rgb(59, 65, 84);
	color: rgb(221, 231, 220);
}

QPushButton::hover {
	border-radius: 10px;
	border-style: solid;
	background-color: rgb(49, 55, 74);
	color: rgb(221, 231, 220);
}</string>
    </property>
    <property name="text">
     <string>jpeg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_gif">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>640</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 10px;
	border-style: solid;
	background-color: rgb(59, 65, 84);
	color: rgb(221, 231, 220);
}

QPushButton::hover {
	border-radius: 10px;
	border-style: solid;
	background-color: rgb(49, 55, 74);
	color: rgb(221, 231, 220);
}</string>
    </property>
    <property name="text">
     <string>gif</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFIle"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
